apps/api/v1/pages/api/connected-calendars/_get.ts

```diff
@@ -0,0 +1,71 @@
+import type { NextApiRequest } from "next";
+
+import type { UserWithCalendars } from "@calcom/lib/getConnectedDestinationCalendars";
+import { getConnectedDestinationCalendars } from "@calcom/lib/getConnectedDestinationCalendars";
+import { HttpError } from "@calcom/lib/http-error";
+import { defaultResponder } from "@calcom/lib/server";
+import prisma from "@calcom/prisma";
+
+import { extractUserIdsFromQuery } from "~/lib/utils/extractUserIdsFromQuery";
+
+/**
+ * @swagger
+ * /connected-calendars:
+ *   get:
+ *     parameters:
+ *       - in: query
+ *         name: apiKey
+ *         required: true
+ *         schema:
+ *           type: string
+ *         description: Your API key
+ *       - in: query
+ *         name: userId
+ *         required: false
+ *         schema:
+ *           type: number
+ *         description: Admins can fetch connected calendars for other user e.g. &userId=1 or multiple users e.g. &userId=1&userId=2
+ *     summary: Fetch connected calendars
+ *     tags:
+ *      - connected-calendars
+ *     responses:
+ *       200:
+ *         description: OK
+ *       401:
+ *        description: Authorization information is missing or invalid.
+ *       403:
+ *        description: Non admin user trying to fetch other user's connected calendars.
+ */
```

Comment:
This is great 👏 
Would be good to have an example response (200) as well 🙏

Reply:
Will update docs with example response once zod response schema is clarified.

Reply:
Done in [feat: swagger doc example response](https://github.com/calcom/cal.com/pull/14176/commits/3769486ad5c7dd54f7b345aeade5896dcad03fb3)